packages/Frontend/src/containers/AuthenticateContainer.jsx

```diff
@@ -23,47 +22,86 @@ class AuthenticateContainer extends Component {
     super(props);
     this.state = {
       email: '',
-      error: '',
+      error: false,
       loading: false,
       password: '',
     };
   }
 
+  componentDidMount = () => {
+    this.setState({ loading: true }, () => {
```

Comment:
This was changed because of how signing in with another provider, like google and facebook, works. We leave this component, go to google/facebook, and then come back to our sign in page where this component is mounted again.  When the component mounts it checks to see if it got to the page from somewhere in our app, or was it redirected here from google/facebook. While that check is running we set loading to true in facebook, when that check is done we set it to false so there's not a loading wheel. In practice, what happens is when you click the login button it does a near instant check and hardly draws a loading wheel, but when you're redirected back from Google it takes quite a while and shows a loading wheel until you're finally sent to the home page.